src/components/Login/Login.tsx

```diff
@@ -10,7 +10,7 @@ import Dropdown from "../Dropdown";
 import * as styles from "./Login.module.scss";
 import * as dropdownStyles from "../Dropdown.module.scss";
 
-const Login = () => {
+const Login = ({ dropDownViewOff }) => {
```

Comment:
I would give this a name that reflects what it does show, vs what it doesn't show and then set it to false by default. For example: 
```javascript
const Login = ({ displayAsButtons = false }) => {
```
If we don't give it a default value, or define an interface for props where this is optional, then we will have typescript errors in the consuming components if they don't pass this prop.

Reply:
Totally make sense ! I adapted it as well ✅